app/controllers/conferences_controller.rb

```diff
@@ -16,12 +16,11 @@ def show
 
   def create
     team = current_user.student_team
```

Comment:
Aha, it is only for orga members and students. Then I still don't get the use case for the change to user_id. Sorry ;-) This controllers#create is were the students enter their 'suggestions', right?
So, why should an orga member use _this_ create action?  Why not either in the namespaced orga/conferences? OR even better, import them via the google database. 
I have the feeling that we are trying to press two very different use cases into one. WDYT?